CRM/Member/Form/MembershipRenewal.php

```diff
@@ -287,6 +291,10 @@ public function buildQuickForm() {
           'financial_type_id' => CRM_Utils_Array::value('financial_type_id', $values),
           'total_amount' => CRM_Utils_Money::format($totalAmount, NULL, '%a'),
           'total_amount_numeric' => $totalAmount,
+          'total_amount_ex_tax' => $totalAmountExTax,
+          'tax_term' => CRM_Utils_Array::value('tax_term', $invoiceSettings),
+          'tax_amount' => $taxAmount,
+          'tax_rate'   => $taxRate,
```

Comment:
Does any of this align with variable names etc. in Form/Membership.php?  Even if we can't consolidate both at this stage we should aim to bring them closer together.

Reply:
I'm pretty keen to see any additional or changed sales tax related code in it's own function - pref shared with other functions